yarn.lock

```diff
@@ -9541,11 +9515,6 @@ caniuse-api@^3.0.0:
     lodash.memoize "^4.1.2"
     lodash.uniq "^4.5.0"
 
-caniuse-db@1.0.30000772:
```

Comment:
I still see this not removed when I run yarn. Can you add this back?

Reply:
You need to explain this more, because I'm just updating packages with yarn add command. What am I doing wrong?

Reply:
I'm not sure what goes wrong here. When I checkout your branch and run yarn without the lock freeze then I see this added back to the lockfile.

Reply:
But drone is not complaining about this, it would if the yarn.lock file needed to update because it runs `yarn install --frozen-lockfile --no-progress`.